backup_and_restore/application_backup_and_restore/installing/about-installing-oadp.adoc

```diff
@@ -37,4 +36,4 @@ You create a default `Secret` and then you install the Data Protection Applicati
 [role="_additional-resources"]
 == Additional resources
 
-* Overview of backup locations and snapshot locations in the link:https://{velero-domain}/docs/v1.7/locations/[Velero documentation].
+* Overview of backup locations and snapshot locations in the link:https://{velero-domain}/docs/v{velero-version}/locations/[Velero documentation].
```

Comment:
We are now supposed to [limit additional resources text](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#additional-resources-sections) to the page title in a link text, like so:
```suggestion
* link:https://{velero-domain}/docs/v{velero-version}/locations/[Velero documentation].
```
You might want to use different text for the link title than what I have in the suggestion box though :)

Reply:
@jeana-redhat 
Please take a look now.

Reply:
Just need to remove the `.` at then end, then LGTM

Feel free to ping me in Slack when it's time to merge (even if it's after my squad shift) so you don't have to wait on another look-over :+1:

Reply:
Done